WPMedia/fusion-news-theme-blocks
# PEN-1432 - Transparent PNGs

## Description

Added logic to apply format png to thumbor URL generation. While this is a fix for only `png` files there is further work to be done on this block to simplify and long term of potentially removing the `format` filter all together.

Added docs for SDK - https://github.com/WPMedia/engine-theme-sdk/pull/186

## Jira Ticket
- [PEN-1432](https://arcpublishing.atlassian.net/browse/PEN-1432)

## Acceptance Criteria

Update Author images in both Short Author Bio and Full Author Bio blocks to use white as the background fill instead of black 

Note: We use white as the background on fallback promo images – you should be able to look there as an example

Update tests/documentation as necessary

## Test Steps

_Expectation of a production core components recent data file locally for testing_

1. In Theme-blocks directory checkout this branch `git checkout PEN-1432-transparent-png-images`
2. In Fusion News Theme start fusion using `npx fusion start -f -l @wpmedia/resizer-image-block`
3. Visit a page that has a transparent png output - http://localhost/pf/author/kt/?_website=the-gazette
4. Verify the authors bio image does not have a black background and is transparent (see after image below)
5. Visit homepage to verify all other images are not affected by this change - http://localhost/pf/homepage/?_website=the-gazette


## Effect Of Changes
### Before

Author image was filled with a black background

<img width="993" alt="PEN-1432-before" src="https://user-images.githubusercontent.com/868127/105698655-13b0f380-5efe-11eb-9e92-7dbe6b979588.png">

### After

Author image is now transparent

<img width="959" alt="PEN-1432-after" src="https://user-images.githubusercontent.com/868127/105698610-009e2380-5efe-11eb-9e90-3f42028ae4d6.png">


## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._

## Reviewer Checklist 
_The reviewer of the PR should copy-paste this template into the review comments on review._
